tests/test_other.py

```diff
@@ -135,6 +135,18 @@ def metafunc(self, backend):
   return metafunc
 
 
+def requires_pkg_config(func):
+  assert callable(func)
+
+  @wraps(func)
+  def decorated(self, *args, **kwargs):
+    if not utils.which('pkg-config'):
```

Comment:
Can you add a `EMTEST_SKIP_PKG_CONFIG` setting (see the existing `EMTEST_SKIP_V8`) so that we can disable these tests in certain environments.